<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MirrorDock</class>
 <widget class="QDockWidget" name="MirrorDock">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>147</width>
    <height>347</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Mirror and Rotate</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>Mirror</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_3">
       <property name="margin">
        <number>2</number>
       </property>
       <item row="0" column="1">
        <widget class="QToolButton" name="mirrorUp">
         <property name="toolTip">
          <string>Mirror up</string>
         </property>
         <property name="text">
          <string>U</string>
         </property>
         <property name="icon">
          <iconset resource="../crochet.qrc">
           <normaloff>:/images/mirror_up.svg</normaloff>:/images/mirror_up.svg</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QToolButton" name="mirrorDown">
         <property name="toolTip">
          <string>Mirror down</string>
         </property>
         <property name="text">
          <string>D</string>
         </property>
         <property name="icon">
          <iconset resource="../crochet.qrc">
           <normaloff>:/images/mirror_down.svg</normaloff>:/images/mirror_down.svg</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QToolButton" name="mirrorLeft">
         <property name="toolTip">
          <string>Mirror left</string>
         </property>
         <property name="text">
          <string>L</string>
         </property>
         <property name="icon">
          <iconset resource="../crochet.qrc">
           <normaloff>:/images/mirror_left.svg</normaloff>:/images/mirror_left.svg</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QToolButton" name="mirrorRight">
         <property name="toolTip">
          <string>Mirror right</string>
         </property>
         <property name="text">
          <string>R</string>
         </property>
         <property name="icon">
          <iconset resource="../crochet.qrc">
           <normaloff>:/images/mirror_right.svg</normaloff>:/images/mirror_right.svg</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="title">
       <string>Rotate</string>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <property name="margin">
        <number>2</number>
       </property>
       <item row="2" column="1">
        <widget class="QToolButton" name="rotate180">
         <property name="toolTip">
          <string>Rotate 180 degrees</string>
         </property>
         <property name="text">
          <string>180</string>
         </property>
         <property name="icon">
          <iconset resource="../crochet.qrc">
           <normaloff>:/images/rotate180.svg</normaloff>:/images/rotate180.svg</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QToolButton" name="rotate270">
         <property name="toolTip">
          <string>Rotate 270 degrees</string>
         </property>
         <property name="text">
          <string>270</string>
         </property>
         <property name="icon">
          <iconset resource="../crochet.qrc">
           <normaloff>:/images/rotate270.svg</normaloff>:/images/rotate270.svg</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QToolButton" name="rotate90">
         <property name="toolTip">
          <string>Rotate 90 degrees</string>
         </property>
         <property name="text">
          <string>90</string>
         </property>
         <property name="icon">
          <iconset resource="../crochet.qrc">
           <normaloff>:/images/rotate90.svg</normaloff>:/images/rotate90.svg</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QToolButton" name="rotateCustom">
         <property name="toolTip">
          <string>Rotate custom degrees</string>
         </property>
         <property name="text">
          <string>C</string>
         </property>
         <property name="iconSize">
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="rotateCustomWidgets" native="true">
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QSpinBox" name="degrees">
         <property name="maximum">
          <number>359</number>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QPushButton" name="rotateBttn">
         <property name="text">
          <string>Rotate</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="../crochet.qrc"/>
 </resources>
 <connections/>
</ui>
